index.js

```diff
@@ -0,0 +1,5 @@
+import BookList from './modules/bookList.js';
+
+const book = new BookList();
+
+console.log(book);
```

Comment:
Kindly remove `console.log` from your js files after the implementation, as it is not a javascript best practice 😄